cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -610,9 +615,14 @@ class ParquetIOTestBase : public ::testing::Test {
   }
 
   void ReaderFromSink(std::unique_ptr<FileReader>* out) {
+    return ReaderFromSink(out, default_arrow_reader_properties());
+  }
+
+  void ReaderFromSink(std::unique_ptr<FileReader>* out,
+                      const ArrowReaderProperties& arrow_reader_properties) {
     ASSERT_OK_AND_ASSIGN(auto buffer, sink_->Finish());
     ASSERT_OK_NO_THROW(OpenFile(std::make_shared<BufferReader>(buffer),
-                                ::arrow::default_memory_pool(), out));
+                                ::arrow::default_memory_pool(), arrow_reader_properties, out));
```

Comment:
I still hold the opinion that we should use `FileReaderBuilder` here instead of adding a new `OpenFile` in the public API.

Reply:
I am so sorry, I made the modifications locally and marked it as resolved, but forgot to upload them.